### PR TITLE
fixed typo : listner -> listener

### PR DESCRIPTION
**Pull request by vincent-fuchs**
## Description

fixed a recurring typo in several places in the code : listner -> listener

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

do we really need a Jira for a typo fix ?

- [X] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [X] Create/update unit or integration tests for verifying the changes made.

- [X] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [X] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [X] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/common/BusinessEventNotificationConstants.java [hunk 1]
```diff
@@ -25,47 +25,55 @@
...
-                                                                                                                                                                                                                                                                                                                        "fixed_deposit_account_create"), SHARE_ACCOUNT_APPROVE(
-                                                                                                                                                                                                                                                                                                                                "share_account_approve"), RECURRING_DEPOSIT_ACCOUNT_CREATE(
-                                                                                                                                                                                                                                                                                                                                        "recurring_deposit_account_create");
+        LOAN_APPROVED("loan_approved"), //
```
Comment (vorburger) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/common/BusinessEventNotificationConstants.java:28]: @vincent-fuchs I would suggest to remove all these trailing `//` empty comments.. but it's too minor to nit pick on and hold back further reviewing and merging this, so I'll ignore it and move ahead. Perhaps you would like to raise a follow-up PR to remove these again, for it to be nicer?

Comment (vincent-fuchs) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/common/BusinessEventNotificationConstants.java:28]: I could do it, but without these, spotless will complain and try to reformat it. Is there another way to tell spotless that it should not reformat ?

**Comment by vorburger** [comment]
@vincent-fuchs I didn't quite understand what the enum reformatting had to do with the listner typo fix, but we are ecastic (super excited) to see your interest in this project, and hope to see more contributions from you, so didn't want to be a pain here and just merge it anyway (but "squashed" it into 1 single commit; hope you don't mind that?). In the future, perhaps you would like to raise separate PR in such cases. We prefer several smaller PRs per specific change, because they are easier for us to code review.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/common/BusinessEventNotificationConstants.java (hunk 1 above)
Comment (vorburger) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/common/BusinessEventNotificationConstants.java:28]: Oh... I see. Didn't realize that. You could try to tweak [our Spotless config](https://github.com/apache/fineract/blob/develop/fineract-provider/build.gradle#L268), it's configured to use [our Eclipse Code Formatter style](https://github.com/apache/fineract/blob/develop/config/fineractdev-formatter.xml), which you could try to change, if this is important to you. FYI @thesmallstar is our resident expert in such matters!

**Comment by vincent-fuchs** [comment]
You are right, there should have been 2 PRs..  sorry, I was a bit lazy ;-)

**Pull request merged**